package.json

```diff
@@ -31,9 +30,9 @@
         "@types/yargs": "^17.0.24",
         "@typescript-eslint/eslint-plugin": "^5.57.1",
         "@typescript-eslint/parser": "^5.57.1",
-        "@zowe/cli": "^7.13.0",
-        "@zowe/cli-test-utils": "^7.13.0",
-        "@zowe/imperative": "^5.12.0",
+        "@zowe/cli": "^7.18.0",
+        "@zowe/cli-test-utils": "^7.18.0",
+        "@zowe/imperative": "^5.18.0",
```

Comment:
not sure if you need both the zowe/cli and zowe/imperative dependencies since the cli exports imperative now, but I could be wrong and could be changed after release.

Reply:
I think I remember have doing that initally when this repo was created but had to go back and bring brack the dependency, @t1m0thyj @zFernand0 do you remember why we decided on bringing this back instead of just importing from the cli?